src/crl.rs

```diff
@@ -560,6 +560,25 @@ impl RevocationReason {
             })?)
         })
     }
+
+    /// Return an iterator over all possible [RevocationReason] variants.
+    pub fn iterator() -> impl Iterator<Item = RevocationReason> {
```

Comment:
Naming nit: I've usually seen this called `iter()`. I think instead of `.iter().copied()` you can just say `.into_iter()`?

Reply:
That makes sense, thanks. Changed to use `iter()` as the name and `into_iter()` for the transformation.